app/templates/direct-award/index.html

```diff
@@ -0,0 +1,71 @@
+{% extends "_base_page.html" %}
+{% import "toolkit/summary-table.html" as summary %}
+{% block page_title %}Saved searches - Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with
+    items = [
+      {
+          "link": url_for('main.index'),
+          "label": "Digital Marketplace"
+      },
+      {
+          "link": "/buyers",
+          "label": "Account Overview"
+      },
+      {
+          "label": "Saved searches"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+<div class="direct-award-saved-searches">
+  <header class="page-heading">
+    <h1>
+      Saved searches
+    </h1>
+    <p class="context">Find, compare and buy cloud services through the G-Cloud framework.</p>
+  </header>
+
+  <div class="grid">
+    
+    {{ summary.heading("Searching", id="searching_table") }}
+    {% call(item) summary.list_table(open_searches,
+      caption="Summary table",
+      empty_message="You have no saved searches",
+      field_headings=[
+        "Name",
+        "Saved"  ],
+      field_headings_visible=True
+    ) %}
+      {% call summary.row() %}
+        {{ summary.service_link(item.name, url_for('direct_award.view_project', framework_framework="g-cloud", project_id=item.id)) }}
+        {{ summary.text(item.createdAt | datetimeformat ) }}
+      {% endcall %}
+    {% endcall %}
+
+    {{ summary.heading("Search Ended", id="search_ended_table") }}
```

Comment:
Along similar lines to the previous comment, can we call this 'Archived searches'?